app/views/obituaries/index.html.erb

```diff
@@ -1,26 +1,26 @@
-<h1>All Obituaries</h1>
-  <table class="table table-striped table-bordered cart-table">
-    <thead>
-      <tr>
-        <th scope="col">Image</th>
-        <th scope="col">First Name</th>
-        <th scope="col">Last Name</th>
-        <th scope="col">Age</th>
-        <th scope="col">City</th>
-        <th scope="col">State</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @obituaries.each do |obituary| %>
-        <tr id="obit-<%= obituary.id %>">
-
-          <td><%= link_to image_tag(obituary.image_url), "/obituaries/#{obituary.id}" %></td>
-          <td><%= obituary.first_name %></td>
-          <td><%= obituary.last_name %></td>
-          <td><%= obituary.age %></td>
-          <td><%= obituary.city %></td>
-          <td><%= obituary.state %></td>
+<div class="container">
+  <h1 class="my-4">Search Results</h1>
+    <table class="table table-sm table-bordered table-striped">
+      <thead class='thead-dark'>
+        <tr>
+          <th scope="col">Image</th>
+          <th scope="col">Name</th>
+          <th scope="col">State</th>
+          <th scope="col">Age</th>
+          <th scope="col">Publication Year</th>
+          <th scope="col">Source</th>
         </tr>
-      <% end %>
-    </tbody>
-  </table>
+      </thead>
+      <tbody style= "background-color:white;">
+        <% @obituaries.each do |obituary| %>
+          <tr id="obit-<%= obituary.id %>">
+            <td><%= link_to image_tag(obituary.image_url, size: "50x50"), "/obituaries/#{obituary.id}" %></td>
+            <td class="align-middle pl-3"><%= obituary.first_name + ' ' + obituary.last_name%></td>
+            <td class="align-middle pl-3"><%= obituary.state %></td>
+            <td class="align-middle pl-3"><%= obituary.age %></td>
+            <td class="align-middle pl-3"><%= "2020" %></td>
+            <td class="align-middle pl-3"><%= "ObituaryHub Community" %></td>
```

Comment:
Hardcoded for display purposes!

Reply:
Great work Styling

Reply:
" + ' '  + " is a much better way to write in a space then the huge gap that I left, will be using this in the future!